test_regress/t/t_randc_oversize_bad.out

```diff
@@ -1,5 +1,5 @@
-%Error: t/t_randc_oversize_bad.v:8:21: Maximum implemented width for randc is 32 bits, 'i' is 38 bits
-                                     : ... note: In instance 't'
-    8 |    randc bit [37:0] i;
```

Comment:
Don't want to loose this error's test.

Reply:
Yeah, I got carried away, when updating tests and noticed this when the error message coverage test failed. It seems to be the consequence of moving this stage to later point after the stage where possible parenthesis-less calls get recognized.

I'm re-invastigating the possibility of running V3Randomize this at an earlier stage.

Reply:
Fixed.